contao/library/Contao/Automator.php

```diff
@@ -607,20 +580,18 @@ public function generateInternalCache()
 	 */
 	public function generateConfigCache()
 	{
+		/** @var KernelInterface $kernel */
 		global $kernel;
 
+		$resources = $kernel->getContainer()->get('contao.resource_provider');
+
 		// Generate the class/template laoder cache file
 		$objCacheFile = new \File('system/cache/config/autoload.php');
 		$objCacheFile->write('<?php '); // add one space to prevent the "unexpected $end" error
 
-		foreach ($kernel->getContaoBundles() as $bundle)
+		foreach ($resources->findFilesIn('config', 'autoload.php') as $file)
```

Comment:
This is still too complicated as @Toflar mentioned in today's call. It should be:

```php
/** @var KernelInterface $kernel */
global $kernel;

$resources = $kernel->getContainer()->get('contao.resource_provider');

// findIn() should only set the base paths and ignoreDotFiles()
$files = $resources->findIn('config')->files()->name($fileName);
```

Especially, we should not call `followLinks()` by default, because there is no way to revert this.